### ISSUE TITLE
AIOKafkaProducer.send_and_wait() throws error when writing to a non-existent topic

### ISSUE THREAD

**Issue by ZmeiGorynych** [aio-libs/aiokafka#95]
When I try to write to a topic that doesn't yet exist, both vanilla kafka-python and AIOKafkaProducer.send() silently create the topic before writing to it, which is how it should be.
However, when I try to do the same in a call to send_and_wait, I get an error, see below:
```
AIOKafkaProducer
async def produce(loop):
    # Just adds message to sending queue
    future = await producer.send(topic+'1', b'some_message_bytes')
    # waiting for message to be delivered
    resp = await future
    print("Message produced: partition {}; offset {}".format(
          resp.partition, resp.offset))
    # Also can use a helper to send and wait in 1 call
    resp = await producer.send_and_wait(
         topic + '2', key=b'foo', value=b'bar')
    #resp = yield from producer.send_and_wait(
    #   'foobar', b'message for partition 1', partition=1)
​
loop = asyncio.get_event_loop()
producer = AIOKafkaProducer(loop=loop, bootstrap_servers=kafka_host, api_version="0.9")
 # Bootstrap client, will get initial cluster metadata
loop.run_until_complete(producer.start())
loop.run_until_complete(produce(loop))
 # Wait for all pending messages to be delivered or expire
loop.run_until_complete(producer.stop())
 #loop.close()
```
Running that produces
```
Message produced: partition 0; offset 3
---------------------------------------------------------------------------
UnknownTopicOrPartitionError              Traceback (most recent call last)
<ipython-input-7-d00f93f5f3ce> in <module>()
     16 # Bootstrap client, will get initial cluster metadata
     17 loop.run_until_complete(producer.start())
---> 18 loop.run_until_complete(produce(loop))
     19 # Wait for all pending messages to be delivered or expire
     20 loop.run_until_complete(producer.stop())

C:\Users\Egor\Anaconda2\envs\py3k\lib\asyncio\base_events.py in run_until_complete(self, future)
    385             raise RuntimeError('Event loop stopped before Future completed.')
    386 
--> 387         return future.result()
    388 
    389     def stop(self):

C:\Users\Egor\Anaconda2\envs\py3k\lib\asyncio\futures.py in result(self)
    272             self._tb_logger = None
    273         if self._exception is not None:
--> 274             raise self._exception
    275         return self._result
    276 

C:\Users\Egor\Anaconda2\envs\py3k\lib\asyncio\tasks.py in _step(***failed resolving arguments***)
    237                 # We use the `send` method directly, because coroutines
    238                 # don't have `__iter__` and `__next__` methods.
--> 239                 result = coro.send(None)
    240             else:
    241                 result = coro.throw(exc)

<ipython-input-7-d00f93f5f3ce> in produce(loop)
      8     # Also can use a helper to send and wait in 1 call
      9     resp = await producer.send_and_wait(
---> 10          topic + '2', key=b'foo', value=b'bar')
     11     #resp = yield from producer.send_and_wait(
     12     #   'foobar', b'message for partition 1', partition=1)

C:\Users\Egor\Anaconda2\envs\py3k\lib\site-packages\aiokafka\producer.py in send_and_wait(self, topic, value, key, partition)
    302     def send_and_wait(self, topic, value=None, key=None, partition=None):
    303         """Publish a message to a topic and wait the result"""
--> 304         future = yield from self.send(topic, value, key, partition)
    305         return (yield from future)
    306 

C:\Users\Egor\Anaconda2\envs\py3k\lib\site-packages\aiokafka\producer.py in send(self, topic, value, key, partition)
    286 
    287         # first make sure the metadata for the topic is available
--> 288         yield from self._wait_on_metadata(topic)
    289 
    290         key_bytes, value_bytes = self._serialize(topic, key, value)

C:\Users\Egor\Anaconda2\envs\py3k\lib\site-packages\aiokafka\producer.py in _wait_on_metadata(self, topic)
    238         yield from self.client.force_metadata_update()
    239         if topic not in self.client.cluster.topics():
--> 240             raise UnknownTopicOrPartitionError()
    241 
    242         return self._metadata.partitions_for_topic(topic)

UnknownTopicOrPartitionError: [Error 3] UnknownTopicOrPartitionError:

**Comment by Drizzt1991** [comment]
Fixed in #96

**Comment by Drizzt1991** [comment]
Should be fixed as of 0.2.0. Closing
